drivers/tapdisk-metrics-stats.h

```diff
@@ -25,15 +25,15 @@
 
 struct stats {
     uint32_t version;
-    unsigned long long oo_reqs;
```

Comment:
You need a uint32_t _pad0; here, or this structure still has different alignment between 32bit and 64bit compiles